web/src/builder.rs

```diff
@@ -574,6 +574,10 @@ impl RuffleInstanceBuilder {
 
         let (renderer, canvas) = self.create_renderer().await?;
 
+        canvas
```

Comment:
I think you can `set_tab_index` for this.

Reply:
You're right! Fixed